WormsApp/worm.cpp

```diff
@@ -28,8 +26,12 @@ void Worm::move() {
   switch (currDir_)
   {
   case LEFT:
-      (x_>0)?--x_:x_%=board_->getWidth();
+      if(x_>0)
+          --x_;
+      else
+          x_%=board_->getWidth();
```

Comment:
This else means that `x_` is negative and I'm not sure whether making modulo (`%`) always makes it positive. Instead I'd do `x_ += board_->getWidth()`.